core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -315,6 +315,8 @@ import scala.util.control.{NoStackTrace, NonFatal}
   override def preStart(): Unit = {
     super.preStart()
     try {
+      if (log.isDebugEnabled)
+        log.debug(settings.toString)
```

Comment:
Some note "creating Kafka consumer with" would be great.

Reply:
@ennru Updated